TEMPLATES/pyproject.toml

```diff
@@ -0,0 +1,2 @@
+[tool.black]
+# line-length = 120
```

Comment:
👎 sorry disagree with such default value. As specified at https://black.readthedocs.io/en/stable/usage_and_configuration/the_basics.html#configuration-format, **The option keys are the same as long names of options on the command line.** And default line-length is 88 ; see in online doc https://megalinter.github.io/descriptors/python_black/#help-content

Reply:
I just realized that line is in comment; anyway, I'd like to see 88 rather than 120 ;-)

Reply:
it's just an empty template ^^

Reply:
Perharps, but personnaly I prefer to present (to all users) that are default linter values. Remember https://github.com/megalinter/megalinter/issues/726

Reply:
That could be changed indeed :)